Robust.Client/GameObjects/Components/Renderable/SpriteComponent.cs

```diff
@@ -296,6 +296,12 @@ void ISerializationHooks.AfterDeserialization()
             LocalMatrix = Matrix3Helpers.CreateTransform(in offset, in rotation, in scale);
         }
 
+        /// <summary>
+        /// Used to allow one to decide if they wish to loop the animation or not.
+        /// </summary>
+        [DataField("loop")]
+        public bool Loop { get; set; } = true; // default true
```

Comment:
```suggestion
        [DataField]
        public bool Loop { get; set; } = true;
```
I'm also not sure if this should be a layer or a sprite property. I.e., what if you want a complicated sprite with some looping animations, and others that only play once?

Reply:
sprite animations aren't handled by the animation system, that's something else. The closest thing to the "animation itself" would be putting it on `RSI.State` and getting it from the RSI's meta.json, and then making the RSI's value the default behavior for a sprite layer